backend/app/main/answer_views.py

```diff
@@ -20,12 +24,68 @@ def answer(question_id):
             question=question,
             session_id=session['id']))
 
+    
+    #Store it in the Airtable
+    if question_id == '1':
+        field_name = "Name"
+    else:
+        field_name = "Email"
+    
+    temp_info = {
+        field_name: request.values['Body'],
+    }
+    temp_field = {"fields": temp_info}
+    temp_json = {"records": [temp_field]}
+
+   # headers = {'Authorization': 'Bearer keybp9BmMeD5OLIjq', 'Content-Type': 'application/json; charset=utf-8'}
```

Comment:
Make sure not to hard-code the Authorization key anywhere, since it can be used by anyone to access our Airtable!  Use the environment variable instead